### PR TITLE
fixed MockData URL on StepDefinitionsEditor

### PR DESCRIPTION
**Pull request by yemo**
1, make changes in build/styles.js to build the src.
2, make changes in MockModel.js to add \ on front of ? into json files
3, make changes in MockParserService.js to display url without all \, that it would not add \ on front of ? every time it saved.

### CODE REVIEW

File: build/styles.js [hunk 1]
```diff
@@ -20,9 +20,9 @@ var errorHandler = require('./utilities/error-handler');
 module.exports = styles;
 
 function styles () {
-    var cssFilter = filter('*.css');
+    var cssFilter = filter('*.css', {restore:true});
```
Comment (phenomnomnominal) [inline build/styles.js]: spacing around object literals: 

``` javascript
var cssFilter = filter('*.css', {
    restore:true
});
```

File: src/app/features/StepDefinitionEditor/Models/MockModel.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ var createMockModelConstructor = function (
         var ast = astCreatorService;
 
         var data = {
-            url: ast.literal(new RegExp(this.url))
+            url: ast.literal(new RegExp(this.url.replace(/\?/g,'\\\?')))
```
Comment (phenomnomnominal) [inline src/app/features/StepDefinitionEditor/Models/MockModel.js]: This is cool, but regexs can be a bit cryptic, especially when escaping stuff. Maybe add some comments as to why we have to do this?

File: src/app/features/StepDefinitionEditor/Services/MockParserService.js [hunk 3]
```diff
@@ -49,10 +49,10 @@ var MockParserService = function MockParserService (
     }
 
     function parseUrl (mock, mockCallExpression) {
-        var url = _.last(mockCallExpression.callee.object.arguments).raw;
-        var urlRegex = new RegExp(url.replace(/^\//, '').replace(/\/$/, ''));
-        assert(urlRegex);
-        return urlRegex.source;
+        var rawUrl = _.last(mockCallExpression.callee.object.arguments).raw;
+        var url = rawUrl.replace(/^\//, '').replace(/\/$/, '').replace(/\\/g,'');
```
Comment (phenomnomnominal) [inline src/app/features/StepDefinitionEditor/Services/MockParserService.js]: Again, maybe add some comments about removing the escaping.

File: build/styles.js (hunk 1 above)
Comment (yemo) [inline build/styles.js]: added spacing for object literals

File: src/app/features/StepDefinitionEditor/Models/MockModel.js (hunk 2 above)
Comment (yemo) [inline src/app/features/StepDefinitionEditor/Models/MockModel.js]: added comments of why we replace ? to \?

File: src/app/features/StepDefinitionEditor/Services/MockParserService.js (hunk 3 above)
Comment (yemo) [inline src/app/features/StepDefinitionEditor/Services/MockParserService.js]: Added comments to explain display url without escaping character from raw url of json file

**Pull request closed without merging**